stanford-sw/src/edu/stanford/StanfordIndexer.java

```diff
@@ -1033,11 +1091,12 @@ else if (tag.startsWith("8"))
     public Set<String> getTopicWithoutTrailingPunct(final Record record, final String fieldSpec, String charsToReplaceRegEx, String charsB4periodRegEx)
     {
       Set<String> resultSet = removeTrailingPunct(record, fieldSpec, charsToReplaceRegEx, charsB4periodRegEx);
-    if (buildings.contains("LANE-MED"))
-      resultSet.removeAll(f655suba);
+    // 655a no longer in topic facet
+    // if (buildings.contains("LANE-MED"))
+    //   resultSet.removeAll(f655suba);
```

Comment:
I would just take these lines out -- it's in the git history.  (when I started writing this code, we were using subversion)